frontend/src/App.tsx

```diff
@@ -6,9 +6,11 @@ import useEquipmentItems from "./hooks/useEquipmentItems";
 import RequireAuth from "./routing/RequireAuth";
 import {ToastContainer} from "react-toastify";
 import LoginPage from "./pages/LoginPage";
+import useAppUser from "./hooks/useAppUser";
 
 function App() {
     const equipmentItems = useEquipmentItems();
+    const appUser = useAppUser();
```

Comment:
Sicherlich wirst Du `appUser` in einem nächsten PR nutzen. Für den Moment ist es aber noch ungenutzt und sollte strenggenommen so nicht nach `main`.

Tipp: gib in so einem Fall einfach eine Eigenschaft des Objekts aus. Dann wird es irgendwie genutzt ;)